docker-compose.yaml

```diff
@@ -31,7 +31,12 @@ services:
   whoami1:
     container_name: whoami1
     image: containous/whoami
+    # container port must be commented out as the traffic is routed through lazytainer
+    #ports:
+    #  - "81:81"
```

Comment:
Are the commented out ports intentional? If so, what is their purpose?

Reply:
In most cases Layzytainer will be deployed to manage containers that are already setup, so it is likely the original container is exposing ports that need to be commented out for Lazytainer to work, so IMO including the commented out ports in the example is more clear

Reply:
Fair points. 

Apologies if I come off as hair-splitting here.
The line you added says "must be commented out"

What if we changed it to 
`# - "81:81" # port should not be configured on this container since traffic is routed through lazytainer`

- conveys the point
- doesn't say you *need* commented code
- line count

Reply:
Yeah you are right, happy with that